Lecture4/style.css

```diff
@@ -0,0 +1,97 @@
+.btn {
+  border-radius: 10px;
+  background-color: rgb(97, 219, 251);
+  color: rgb(28, 2, 60);
+}
+
+.search {
+  outline: auto;
+  outline-color: rgb(97, 219, 251);
+}
+
+.navbar {
+  background-color: rgb(169, 169, 169) !important;
+  /* background-image: linear-gradient(
+    to right,
+    rgb(28, 2, 60),
+    rgb(126, 51, 237),
+    rgb(28, 2, 60)
+  ); */
+  color: #fff !important;
+}
+
+a {
+  color: rgb(243, 221, 253) !important;
+}
+
+.card-img-top {
+  margin-right: 1rem;
+  border-radius: 15px 15px 0 0px !important;
+}
+
+.img-fluid {
+  min-height: 180px;
+  max-height: 180px;
+  min-width: 10.9rem;
+  max-width: 10.9rem;
+}
+
+.card {
+  border-radius: 15px !important;
+  height: 320px;
+  /* background: linear-gradient(to right, #03a9f4, #fff 100%); */
+  backdrop-filter: blur(15px);
+  box-shadow: 5px 6px 16px #000000;
+  /* border-top: rgb(28, 2, 60) 1px solid;
+  border-left: rgb(28, 2, 60) 1px solid; */
+}
+
+.card-footer {
+  padding: 0;
+  border-radius: 0px 0 15px 15px !important;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+body {
```

Comment:
Do the body, * and html tags styling at the top so that CSS specificity will not hamper if styles through class and id selectors are given.

Reply:
Okay I agree will make the changes